airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -427,33 +455,79 @@ definitions:
         type: string
         enum: [OAuthAuthenticator]
       client_id:
+        title: Client ID
+        description: The OAuth client ID
         type: string
+        examples:
+          - "{{ config['client_id }}"
+          - "{{ config['credentials']['client_id }}"
       client_secret:
+        title: Client Secret
+        description: The OAuth client secret
         type: string
+        examples:
+          - "{{ config['client_secret }}"
+          - "{{ config['credentials']['client_secret }}"
       refresh_token:
+        title: Refresh Token
+        description: Credential artifact used to get a new access token
         type: string
+        examples:
+          - "{{ config['refresh_token'] }}"
+          - "{{ config['credentials]['refresh_token'] }}"
       token_refresh_endpoint:
+        title: Token Refresh Endpoint
+        description: Endpoint to send a POST request in order to get a new access token
         type: string
+        examples:
+          - https://connect.squareup.com/oauth2/token
```

Comment:
A random example I pulled. Happy to change it if we want to standardize the examples to the API we'll use in the tutorials and concept docs